jraph/_src/utils.py

```diff
@@ -15,14 +15,17 @@
 """Utilities for working with `GraphsTuple`s."""
 
 import functools
-from typing import Any, Callable, Iterable, List, Mapping, Optional, Sequence, Union
+from typing import Any, Callable, Iterable, List, Mapping, Optional, Sequence, \
+  Union
 
 import jax
 import jax.numpy as jnp
+import jax.ops
 import jax.tree_util as tree
-from jraph._src import graph as gn_graph
 import numpy as np
+from jax import lax
 
+from jraph._src import graph as gn_graph
```

Comment:
Why have you changed the ordering?

Reply:
No meaning, the editor I use "cleans up" imports automatically. Will revert.